Project2015To2017/ProjectConverter.cs

```diff
@@ -177,5 +157,27 @@ internal static bool Validate(FileInfo file, ILogger logger)
 			logger.LogError($"File {file.FullName} could not be found.");
 			return false;
 		}
+
+		private static IReadOnlyCollection<ITransformation> TransformationsToApply(ConversionOptions conversionOptions, ILogger logger)
+		{
+			return new ITransformation[]
```

Comment:
Hmm. That would cause merge conflicts with #170.

Reply:
Yes there will be merge conflicts.